pkg/domain/infra/abi/images.go

```diff
@@ -718,7 +719,7 @@ func (ir *ImageEngine) Sign(ctx context.Context, names []string, options entitie
 					logrus.Errorf("unable to close %s image source %q", srcRef.DockerReference().Name(), err)
 				}
 			}()
-			getManifest, _, err := rawSource.GetManifest(ctx, nil)
+			getManifest, manifestType, err := rawSource.GetManifest(ctx, nil)
```

Comment:
(Unrelated: `s/getManifest/manifestBlob/` , or maybe  …`/topManifest[Blob]/`, would be more readable.)